.github/workflows/main.yml

```diff
@@ -49,19 +58,49 @@ jobs:
             libvorbis-dev \
             libspdlog-dev \
             libglew-dev \
-            libgtest-dev
+            libgtest-dev \
+            ccache
 
-      - name: Install dependencies with vcpkg
+      - name: Override cache folder for vcpkg
```

Comment:
Maybe you can reword this step slightly. From what I understand, you are generating the log of the vcpkg installation to hash it for having a unique identifier for the cache.